acceptance_modelisation/radial_acceptance_map_creator.py

```diff
@@ -44,10 +45,12 @@ def __init__(self,
             Initial bin size for cos zenith binning
         max_angular_separation_wobble : u.Quantity, optional
             The maximum angular separation between identified wobbles, in degrees
+        zenith_binning_run_splitting : flaot, optional
+            If true, will split each run to match zenith binning for the base model computation, no effect on the interpolation phase itself, could be compute expensive
         max_fraction_pixel_rotation_fov : float, optional
             For camera frame transformation the maximum size relative to a pixel a rotation is allowed
-        time_resolution_rotation_fov : astropy.unit.Quantity, optional
-            Time resolution to use for the computation of the rotation of the FoV
+        time_resolution_run_splitting : astropy.units.Quantity, optional
+            Time resolution to use for the computation of the rotation of the FoV and cut as function of the zenith bins
```

Comment:
Maybe the name is misleading if it can be used for FoV rotation without the run split